src/utils/style-formatter.js

```diff
@@ -13,15 +13,12 @@ export default {
   getStyles(style, Theme) {
     // TODO: use constants for default values?
     let styles = 'width: 100%;height: 100%;font-weight: bold;';
-
-    if (style) {
+    if (Theme) {
```

Comment:
Should add a test for when Theme is undefined, given that could actually happen, otherwise remove the check.

Reply:
Also `Theme && (palette = Theme.getCurrent().properties.palettes.ui[0].colors)` might look nicer

Reply:
wrap the theme stuff in its own file as well, there is a chance that that palettes doesn't exist on the theme properties, so we need a way to fall back to something in that case. And for the case where Theme doesn't exist at all, like when outside of sense.